joplin/base/wagtail_hooks.py

```diff
@@ -69,7 +69,7 @@ def joplin_page_listing_buttons(page, page_perms, is_parent=False):
             attrs={'title': _("Preview draft version of '{title}'").format(title=page.get_admin_display_title()), 'target': '_blank'},
             priority=20
         )
-    if page.live and page.url:
+    if page.live and page.url and hasattr(page, 'janis_url'):
```

Comment:
Could we leave a comment here about what `janis_url` is and what its being used for in this context?